src/lazy/text/raw/v1_1/reader.rs

```diff
@@ -168,13 +168,29 @@ impl<'top> RawTextListIterator_1_1<'top> {
             has_returned_error: false,
         }
     }
+}
 
-    pub(crate) fn find_span(&self) -> IonResult<Range<usize>> {
+pub(crate) struct TextListSpanFinder_1_1<'top> {
+    pub(crate) allocator: &'top bumpalo::Bump,
+    pub(crate) iterator: RawTextListIterator_1_1<'top>,
+}
+
+impl<'top> TextListSpanFinder_1_1<'top> {
+    pub(crate) fn find_span(
```

Comment:
As far as I can tell, this new function isn't called anywhere, so I'm having a hard time understanding how the caching actually helps anything.